mod.ts

```diff
@@ -127,16 +127,6 @@ export class AWSSignerV4 implements Signer {
     return new Request(request.url, {
       headers,
       method: request.method,
-      body,
```

Comment:
You can not drop the body here. Having the body is important.

Reply:
I know what you are trying to mitigate with this. However the real fix is to update line 98 to `: null;`.